aten/src/ATen/native/sparse/cuda/SparseCsrTensorMath.cu

```diff
@@ -736,4 +814,13 @@ Tensor _sparse_csr_prod_cuda(const Tensor& input, IntArrayRef dims_to_reduce, bo
   return result;
 }
 
+Tensor _sparse_csr_linear_solve(  const Tensor& A,
+  const Tensor& b,
+  const bool left) {
+    Tensor out = b.new_empty(b.sizes());
```

Comment:
Strides of `b` should be checked if they are compatible with `_apply_sparse_csr_linear_solve` if not a copy into a column-major contiguous tensor needs to be done.

Reply:
Updated in [fed22ad](https://github.com/pytorch/pytorch/pull/129856/commits/fed22ad2a3b533948b5f6ed779f0fa0a58eeb0de)